site-service/src/main/resources/site-configuration.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
shouldn't this include some validation on the parent? Please check if there's a format for FHIR references for other orgs (for instance, this is probably a guid, but you should confirm)